controllers/v1/observationsController.js

```diff
@@ -539,7 +538,16 @@ module.exports = class Observations extends Abstract {
                     }
                 }
 
-                let entityDocuments = await entitiesHelper.search(observationDocument.entityTypeId, req.searchText, req.pageSize, req.pageNo);
+                let entityDocuments = 
+                await entitiesHelper.search(
+                    observationDocument.entityTypeId, 
+                    req.searchText, 
+                    req.pageSize, 
+                    req.pageNo,
+                    false,
+                    req.query.schoolTypes,
```

Comment:
@aman-tunerlabs - We should not have schoolTypes and administrationTypes coming from query or request. It is taken from the logged in user profile